test/integration/hybrid_amazon_linux/controls/default.rb

```diff
@@ -40,7 +40,7 @@
 
   post("#{api}/services", { 'name' => 'httpbin', 'url' => 'http://httpbin.org' })
   post("#{api}/services/httpbin/routes", { 'name' => 'httpbin', 'paths' => '/httpbin' })
-  sleep(10) # wait for route to propagate
+  sleep(20) # wait for route to propagate
```

Comment:
what was the reason for increasing the sleep?

Reply:
it seems race issue between httpbin route to propagate and kitchen test

Reply:
20 seconds is a long time to propagate a route, maybe the issue is something else. Are you able to replicate the issue and show me?

Reply:
httpbin route propagate and kitchen test is not always failing and may be wait method needs to added like other kitcken tests.